cocotb_test/simulator.py

```diff
@@ -239,16 +244,24 @@ def get_define_commands(self, defines):
     def get_parameter_commands(self, parameters):
         raise NotImplementedError()
 
-    def get_abs_paths(self, paths):
+    def normalize_paths_list(self, paths):
```

Comment:
```suggestion
    def normalize_paths(self, paths):
```
Because return value could be any type as long as you can iterate over it, right?